src/dlls/mscoree/unixinterface.cpp

```diff
@@ -146,6 +138,61 @@ int coreclr_initialize(
     ConstWStringArrayHolder propertyValuesW;
     propertyValuesW.Set(StringArrayToUnicode(propertyCount, propertyValues), propertyCount);
 
+    // Process properties that specify CoreCLR startup flags and remove them from the property list
+    STARTUP_FLAGS startupFlags =
+        static_cast<STARTUP_FLAGS>(
+            STARTUP_FLAGS::STARTUP_LOADER_OPTIMIZATION_SINGLE_DOMAIN |
+            STARTUP_FLAGS::STARTUP_SINGLE_APPDOMAIN);
+    int newPropertyIndex = 0;
+    for (int oldPropertyIndex = 0; oldPropertyIndex < propertyCount; ++oldPropertyIndex)
+    {
+        if (wcscmp(propertyKeysW[oldPropertyIndex], W("CONCURRENT_GC")) == 0)
+        {
+            if (wcscmp(propertyValuesW[oldPropertyIndex], W("1")) == 0)
+            {
+                startupFlags = static_cast<STARTUP_FLAGS>(startupFlags | STARTUP_CONCURRENT_GC);
```

Comment:
In the legacy hosting interface it looks like flags can only be set but not unset. Are you suggesting making concurrent GC the default only for Unix, or should I make it the default on Windows too add an API to unset flags?